public/js/redux/chartsRedux.js

```diff
@@ -28,9 +30,15 @@ const chartsRedux = State({
         }
     },
 
-    updateComposerVsIncopy(state, composerVsInCopyData) {
+    updateComposerVsIncopy(state, { chartData, startDate, endDate }) {
+        const { composerResponse, inCopyResponse } = chartData;
+        const range = endDate.diff(startDate, 'days');
+        const composerData = composerResponse.data.length < range ?  fillMissingDates(startDate, endDate, composerResponse.data).sort(compareDates) : composerResponse.data;
```

Comment:
Sorry, that wasn't clear, I actually meant in API JS file so `response.then(responseCleanerAndFormatter).then(updateComposerVsIncopyOrWhatever)`.

But I also believe we're doing a lot of formatting for this specific client in the API right now so adding this part of it to the actual API response (from the backend) would make more sense *or* completely making that generic and formatting it totally for this JS app in the frontend, that's probably a larger refactoring task outside of this PR though.